scripts/release.sh

```diff
@@ -97,7 +97,7 @@ yq -i e '.version = "'${newVersion}'"' example/pubspec.yaml
 sed -i '' -e "s/$version/$newVersion/g" ios/appcues_flutter.podspec
```

Comment:
Does this one need to potentially be updated to be less aggressive as well?

Reply:
i think that would be wise, yes. will do, thanks.

Reply:
updated @mmaatttt